### PR TITLE
Feature/color internationalization kotlin (lets see if Jenkins complains)

### PR DESCRIPTION
**Pull request by TortleWortle**
# Pull request
This is mainly to see if jenkins trips.
 - [x] I have read and agreed to the [code of conduct](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CONTRIBUTING.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [GitHub projects](https://github.com/orgs/CascadeBot/projects).
 - [x] I have tested all of my changes.
 
## Added/Changed
Color internationalization and kotlin

## Feature
Kotlin

### CODE REVIEW

File: src/main/java/org/cascadebot/cascadebot/utils/ColorUtils.kt [hunk 1]
```diff
@@ -0,0 +1,126 @@
+package org.cascadebot.cascadebot.utils
```
Comment (binaryoverload) [inline src/main/java/org/cascadebot/cascadebot/utils/ColorUtils.kt]: afaik Kotlin files should be under src/main/kotlin

File: pom.xml [hunk 2]
```diff
@@ -13,20 +13,34 @@
...
+                        <goals>
+                            <goal>test-compile</goal>
+                        </goals>
+                    </execution>
+                </executions>
                 <configuration>
-                    <source>11</source>
-                    <target>11</target>
-                    <release>11</release>
+                    <jvmTarget>1.8</jvmTarget>
```
Comment (binaryoverload) [inline pom.xml]: 1.8? We run Java 11 👀

File: src/main/java/org/cascadebot/cascadebot/utils/ColorUtils.kt (hunk 1 above)
Comment (TortleWortle) [reply src/main/java/org/cascadebot/cascadebot/utils/ColorUtils.kt]: Not necessarily, they can be in src/main/java

File: pom.xml (hunk 2 above)
Comment (TortleWortle) [reply pom.xml]: I figured it wouldn't make much of a difference just for testing if jenkins was gonna work or not, I'll change it

**Comment by weeryan17** [comment]
Closing this at ducks request as this was mainly a test of kotlin with jenkins

**Pull request closed without merging**